### ISSUE TITLE
create docs/ directory with pkgdown

### ISSUE THREAD

**Issue by sgibb** [lgatto/synapter#132]
Depends #123, #124.

**Comment by lgatto** [comment]
This will be done once everything is merged into master.

**Comment by sgibb** [comment]
@lgatto I generated the /docs directory with `pkgdown`. Could you please enable GithubPages for `synapter` and put the URL into the repository description.

**Comment by sgibb** [comment]
@lgatto thanks! Could you please add https://lgatto.github.io/synapter/ to the description of the repository on github? Or do we want to put it into the `README.md`?

**Comment by lgatto** [comment]
All done.
